cli/errors.ts

```diff
@@ -9,6 +9,13 @@ export function isCodaError(value: any): value is CodaError {
   return value && 'statusCode' in value && typeof value.statusCode === 'number' && value.statusCode >= 400;
 }
 
+export function tryParseSystemError(error: any) {
+  if (error.errno === 'UNABLE_TO_VERIFY_LEAF_SIGNATURE') {
+    return 'Run `export NODE_TLS_REJECT_UNAUTHORIZED=1` and rerun your command.';
```

Comment:
Would add a note that this should only apply to Coda developers working in a local development environment (this is all open-source now so we should be clear when we're talking to Codans and when we're talking to external developers who may be looking at how the SDK is implemented).